resources/platforms.json

```diff
@@ -3940,6 +3940,15 @@
         "Platform_Version": "#MAJORVER#.#MINORVER#"
       }
     },
+    "Android_9_0": {
+      "match": "*Linux*Android?9*",
```

Comment:
Should we make sure there's a semicolon after the `9` here to weed out the spoofs (as discussed in #1896)?

Reply:
I'll change this.

Reply:
changed